lib/core/output_sink/es_sink.rb

```diff
@@ -58,6 +56,8 @@ def delete_multiple(ids)
 
     private
 
+    attr_accessor :index_name
```

Comment:
Why not keep it public? Since we're developing this for public consumption, I am inclined to think we should err towards public access, but that's a loosely held intuition, and I'm more just curious about your motivation.

Reply:
I think we can make it public but `attr_reader`

Reply:
I don't actually see any place using the `index_name` outside of the `EsSink` itself, so it's just a cleanup. Also since there's a base class `BaseSink`, I think it makes sense to keep the same public facing interface as the other `BaseSink` implementation - it might not be the case for us if we _need_ to access the `index_name` from outside of sink, but it's already available when you instantiate it - see `initialize` method!